### PR TITLE
Register UI bisher ohne Logik

### PR DESCRIPTION
**Pull request by PhillippPatzelt**

### CODE REVIEW

File: app/src/main/res/drawable/placeholder_logo.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- drawable/ellipse_1.xml -->
+<!-- drawable/placeholder.xml -->
```
Comment (NicoB-Code) [inline app/src/main/res/drawable/placeholder_logo.xml]: Consider replacing the "placeholder" with something like logo_prototype.xml. Yes i know, that we do not have a logo yet, but at least set a TO-DO mark, just as a reminder.

**Review by NicoB-Code — CHANGES_REQUESTED** [review]
One small little detail, otherwise LGTM

File: app/src/main/res/drawable/placeholder_logo.xml (hunk 1 above)
Comment (PhillippPatzelt) [reply app/src/main/res/drawable/placeholder_logo.xml]: Done

**Review by NicoB-Code — APPROVED** [review]
👍

**Pull request merged**